.github/workflows/push_attestation_image.yml

```diff
@@ -46,7 +46,9 @@ jobs:
           if [ ${{ github.event_name }} == "push" ]; then
             echo "branch_name=main" >> $GITHUB_OUTPUT
           else  
-            echo "branch_name=${{ github.head_ref }}" >> $GITHUB_OUTPUT
+            branch_name=${{ github.head_ref }}
+            branch_name=$(echo $branch_name | sed 's/[^a-zA-Z0-9]/-/g')
```

Comment:
From the code, it might be hard to see why this sed is necessary. So how about using a new variable image_name here?
`image_name=$(echo $branch_name | sed 's/[^a-zA-Z0-9]/-/g')`

Reply:
https://github.com/microsoft/confidential-sidecar-containers/pull/32/commits/67cd179d2c29f504acc713ce8af4656c9f5456c6